### ISSUE TITLE
Camera Upload Repeatative

### ISSUE THREAD

**Issue by magedassad** [meganz/android#27]
There isn't a way to keep camera uploads on without uploading old ones. 
Also, if I were to delete the pictures that I no longer want, Mega automatically re-uploads them. Used 3GB on this issue. I don't want it to sync my camera phone on my phone with my account, I just need it to upload one.

**Comment by eegh9Od2ahNg9aej** [comment]
Agreed. Uploading as a function of backing up which is what I got Mega for (not cloud sync, cloud backup). My issues is closely linked to the above. Sometimes, it re-uploads all my camera pictures stored locally on my phone. It seems to detect at some point that, although I have deleted pictures in the cloud, they are not in sync anymore. I don't always keep the pictures that are uploaded, so maybe a setting where it asks to restart the sync or only apply to new would be a good feature.
